src/test/java/uk/gov/ida/eventemitter/TestEvent.java

```diff
@@ -30,4 +32,29 @@ public DateTime getTimestamp() {
     public String getEventType() {
         return eventType;
     }
+
+    @Override
+    public String toString() {
+        final StringBuffer sb = new StringBuffer("TestEvent{");
+        sb.append("eventId=").append(eventId);
+        sb.append(", timestamp=").append(timestamp);
+        sb.append(", eventType='").append(eventType).append('\'');
+        sb.append('}');
+        return sb.toString();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
I think we normally prefer not to use inline conditional and return statements (though I could be wrong)

Reply:
Yes, that's correct. I have made the changes.